app/queries/get_content_item_ids_with_fallbacks.rb

```diff
@@ -0,0 +1,55 @@
+module Queries
+  class GetContentItemIdsWithFallbacks
+    extend ArelHelpers
+
+    class << self
+      def call(content_ids, locale_fallback_order: ContentItem::DEFAULT_LOCALE, state_fallback_order:)
+        state_fallback_order = Array(state_fallback_order).map(&:to_s)
+        locale_fallback_order = Array(locale_fallback_order).map(&:to_s)
+
+        fallbacks = cte(
+          table(:content_items).project(
```

Comment:
repeated calls to `table(:foo)` make this a bit verbose. Maybe use local variables/methods?